benchbuild/experiment.py

```diff
@@ -46,25 +46,17 @@ class HelloExperiment(Experiment):
 Projects = tp.List[ProjectT]
 
 
-class ExperimentRegistry(type):
-    """Registry for benchbuild experiments."""
+class NamedEntity:  # pylint: disable=too-few-public-methods
 
-    experiments = {}
+    @classmethod
+    def __init_subclass__(cls, *args: tp.Any, entity_name: str, **kwargs):
```

Comment:
```suggestion
    def __init_subclass__(cls, *args: tp.Any, entity_name: str, **kwargs: tp.Any):
```
You can be more specific here but some form of type hint is needed.

Reply:
Oh yea, missed it.